src/Files.App/Utils/RecycleBin/RecycleBinHelpers.cs

```diff
@@ -92,7 +92,10 @@ public static async Task RestoreRecycleBinAsync()
 
 			if (result == ContentDialogResult.Primary)
 			{
-				Vanara.Windows.Shell.RecycleBin.RestoreAll();
+				SafetyExtensions.IgnoreExceptions(() =>
```

Comment:
This should work, but wouldn't it be more helpful to have a dialog if an exception occurs?

Reply:
Added 👍